exercise1.Rmd

```diff
@@ -13,4 +13,41 @@ knitr::opts_chunk$set(
   collapse = TRUE,
   comment = "#>"
 )
+
+library(ggplot2)
+ba = read.table("https://raw.githubusercontent.com/OxfordIHTM/teaching_datasets/main/ba.dat",header=TRUE)
+
+average = rowMeans(ba)
+ba$average = average
+
+#Mean differences 
+difference <- ba$Wright - ba$Mini
+ba$difference = difference
+mean_difference = mean(difference)
+
+#Confidence interval
+lower <- mean_difference - 1.96*sd(ba$difference)
+upper <- mean_difference + 1.96*sd(ba$difference)
 ```
+
+# Task 4: Report 
+
+## 1. Table of the mean and difference of per subject measurements made by the Wright and the Mini-Wright
+
+
+
+## 2. Bland and Altman plot
+```{r plot}
+plot = ggplot(ba, aes(x = average, y = difference)) +
+  geom_point() +
+  geom_hline(yintercept = mean_difference) +
+  geom_hline(yintercept = lower, color = "red", linetype="dashed") +
+  geom_hline(yintercept = upper, color = "red", linetype="dashed") +
+  ylab("PEFR (l/min) Mini Wright meter - Wright meter ") +
+  xlab("PEFR (l/min) Mini Wright meter + Wright meter / 2")
+
+plot 
```

Comment:
Good stuff! Good use of the code chunk and placement of plotting code inside.

My only main feedback and this is rather important so I will have to ask you to edit this before I approve it.

Here, you assigned the output of your ggplot script into an object called `plot`. The word `plot` is also the name of a function in R called `plot`. So, whilst the code will not cause an error, any code after you run these lines that needs/uses the `plot` function will not work properly anymore because within your code, the environment has changed and that `plot` is not a function anymore but an plotting object.

Generally, we try to avoid using a name of an existing function as an object name to store outputs. So I would suggest renaming this to maybe `ba_plot` instead.

Does this make sense?